pegasus/sites.v3/code.org/public/css/brstrap.css

```diff
@@ -48,12 +48,16 @@
   .col-23 { width: 23%; float:left; }
```

Comment:
Do you know why we have a bunch of slightly different column widths? How pervasive are these and how much of a difference does it actually make? Can we make them more standardized?